Makefile

```diff
@@ -3,7 +3,7 @@ SHELL := /bin/bash
 ifeq ($(TEST_SERVER_MODE), true)
 	# exclude test_kinesisvideoarchivedmedia
 	# because testing with moto_server is difficult with data-endpoint
-	TEST_EXCLUDE := -k 'not (test_kinesisvideoarchivedmedia or test_awslambda or test_batch or test_ec2 or test_sqs)'
+	TEST_EXCLUDE := -k 'not (test_kinesisvideoarchivedmedia or test_awslambda or test_batch or test_ec2 or test_sqs or test_context_manager)'
```

Comment:
I think I'm understanding the reason for this failure correctly - are you comfortable skipping the test in server mode?

https://github.com/spulec/moto/pull/4596/commits/4d87f3c786500fa777630da79c8c1873486e9e41

> The test tries to list SQS queues outside of the scope of the context
> manager to prove that the context manager exited correctly.
> 
> But when running in server mode, this call still succeeds because the
> server is still running.